src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -246,6 +246,16 @@ To set attributes for the HTML `body` tag use the `<attribute>` instruction.
 </page>
 ```
 
+Also from admin site __Layout Update XML__ field of __Design__ section on _Content Edit_ page  set attributes for the HTML `body` tag use the `<attribute>` instruction.
+
+**Example:** Add a new class to the `body` tag.
+
+```xml
+    <body>
+        <attribute name="class" value="my-new-body-class"/>
+    </body>
+```
+
```

Comment:
This example is already provided on lines [229-237](https://github.com/magento/devdocs/pull/6989/files#diff-627f282353dc7acf09477a0c1d604707L231-R237).

Reply:
Delete this example. It duplicates the example that appears on 229-237:

**Example:** Add a new class to the `body` tag.
```xml
<page>
    <body>
        <attribute name="class" value="my-new-body-class"/>
    </body>
</page>
```

Reply:
Still need to delete this example -- Make sure you check for linting errors -- no trailing spaces or extra blank lines after deleting.